src/app/app/app-layout/editor-footer/editor-footer.component.ts

```diff
@@ -44,4 +52,14 @@ export class EditorFooterComponent {
     clearLogs() {
         this.editorFooterService.clearLogs();
     }
+
+    changeFilter() {
```

Comment:
This whole logic can be moved down into a selector, which can be unit tested separately apart from this componen t.
This selector accepts a LogMessageInitiator parameter and depending on it, filters the result. But most importantly not by the display name!! This is only for ui.
The `key` is the property which we should compare agains and later maybe we can consider the `extra` as well (this is why the extra is there for, if we want to create more complex filters)